cmd/svcat/class/get_cmd.go

```diff
@@ -82,7 +94,13 @@ func (c *getCmd) Run() error {
 }
 
 func (c *getCmd) getAll() error {
-	classes, err := c.App.RetrieveClasses()
+	var opts *servicecatalog.FilterOptions
+
+	opts = &servicecatalog.FilterOptions{
```

Comment:
I like having the options in a struct instead of a bunch of extra parameters 💯

Reply:
FYI, in Go you can define `opts` in a single line like this:

```
opts := &servicecatalog.FilterOptions{...}
```

Instead of declaring a var and then immediately setting it.